index.html

```diff
@@ -30,38 +30,60 @@ <h2 class="project-tagline">Experimental plugin demos with Flocker, Weave, Docke
 
     </section>
 
-    <section class="main-content">
-      <h3>Here are some experimental demos using Flocker and Weave as Docker plugins with Docker Swarm and Docker Compose</h3>
+    <section class="cinema">
+      <div style="max-height: 315px; max-width: 560px; margin: auto; text-align: initial;">
+        <div class='embed-container' style="max-height: 315px;">
+          <iframe src='http://www.youtube.com/embed/QILiHiTD3uc' frameborder='0' allowfullscreen></iframe>
+        </div>
+      </div>
 
-<p>This website and repo will allow you to play with a Vagrant cluster in the following configuration.</p>
+      <h3>Learn More</h3>
+        <a class="btn" href="#">
+          Flocker plugin info
+        </a>
+        &nbsp;
+        <a class="btn" href="#">
+          Weave plugin info
+        </a>
+        &nbsp;
+        <a class="btn" href="#try">
+          Try it yourself now!
+        </a>
 
-<h4>Prerequisites</h4>
-<ul>
-    <li><a href="http://www.vagrantup.com/downloads.html">Vagrant 1.7.2</a></li>
-</ul>
+    </section>
 
-<pre><code>$ git clone https://github.com/plugin-demos-2015/demo
-$ cd demo
-$ make -C vagrantrunner
-</code></pre>
+    <section class="main-content" id="try">
 
-<h3>Who's behind this?</h3>
+      <p style="font-size: larger;">
+        There two way to reproduce the demo, it's quickest to use our infrastructure (it will take literally few minutes to spin up),
+        but you might wish to recreate it on your own using Vagrant also.
```

Comment:
I don't know if I'll be able to help, I need to go to sleep soon - got a big day tomorrow. Where are you guys?